internal/version/version.go

```diff
@@ -44,11 +48,8 @@ var (
 // the official release process (as opposed to a user just running "go build",
 // or running in a CI environment, or something else).
 func IsReleaseBuild() bool {
-	// "source" is a sentinel value returned from buildinfo.Version() when the
-	// build metadata is not present. The build metadata is filled in with the
-	// "-X" flag during release builds, so if it's not present, this binary is
-	// not a release build.
-	return Version != "source"
+	// Binary from release build should follow semver format.
+	return semver.IsValid("v" + Version)
```

Comment:
I googled it. Looks like the Version should not have `v` prefix by default. Let me know if I'm wrong

Reply:
But I saw some conflicts, when I run `abc templates -version` on v0.0.5, it shows `abc v0.5.0 (HEAD, darwin/amd64)` from `HumanVersion = Name + " " + Version + " (" + Commit + ", " + OSArch + ")"`, looks like `Version` has v prefix.

But for https://github.com/abcxyz/abc/blob/3d99e9bae57b99bc551904ab1b916ecd3c9c4118/.goreleaser.yaml#L50, and the 
<img width="572" alt="Screenshot 2024-02-14 at 9 25 54 PM" src="https://github.com/abcxyz/abc/assets/52932582/daa2340d-c796-49dd-860c-36a866e13ea3">, the `Version` doesn't have v prefix